### DISCUSSION TITLE
Minimum postgres requirement

### DISCUSSION THREAD

**Discussion by hmoffatt** [semaphoreui/semaphore#1747] [Q&A] [answered]
Is there a minimum postgres version required? The docker-compose example in the documentation uses 14, but I am using 13 on my host and it seems to be working fine.

**Comment by tboerger** [comment, marked as answer]
I don't think that there are hard requirements as there are no specific postgres features used within semaphore.

**Reply by hmoffatt** [reply]
Thanks, I figured this would be the case.
